//: [Назад](@previous)
import Foundation
//: ## Try / Catch

//: Для упрощения работы с исключениями в **Swift** существует такой инструмент как - **throws**
//: Все функции помеченные ключевым словом **throws** обязаны возвращать *исключение* лишая тем самым функцию возможностей к непредвиденному завершению приложения.
//: > **Исключения** это ошибки имплементирующие протокол **Error**
//: По сути своей мы просто обрабатываем все возможные ошибки в *guard'e* с использованием **throws**.


//: ### Домашнее задание
//: Опробуйте в деле возможности Try / Catch

//: [Вперёд](@next)
